yt_dlp/extractor/startrek.py

```diff
@@ -0,0 +1,59 @@
+from yt_dlp.utils import int_or_none
+from .common import InfoExtractor
+
+
+class StarTrekIE(InfoExtractor):
+    _VALID_URL = r'(?P<base>https://(?:intl|www)\.startrek\.com)/videos/(?P<id>[^/]+)'
+    _TESTS = [{
+        'url': 'https://intl.startrek.com/videos/watch-welcoming-jess-bush-to-the-ready-room',
+        'md5': '491df5035c9d4dc7f63c79caaf9c839e',
+        'info_dict': {
+            'id': 'watch-welcoming-jess-bush-to-the-ready-room',
+            'ext': 'mp4',
+            'title': 'WATCH: Welcoming Jess Bush to The Ready Room',
+            'duration': 1888,
+            'timestamp': 1655388000,
+            'upload_date': '20220616',
+            'description': 'md5:1ffee884e3920afbdd6dd04e926a1221',
+            # Thumbnail seems to have some tracking parameters added, ignore those.
+            'thumbnail': r're:https://intl\.startrek\.com/sites/default/files/styles/video_1920x1080/public/images/2022-06/pp_14794_rr_thumb_107_yt_16x9.jpg(?:\?.+)?',
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        urlbase = self._match_valid_url(url).group('base')
```

Comment:
How about something like this
```
    video_id, urlbase = self._match_valid_url(url).group('id', 'base')
```

Reply:
Oh nice, I didn’t know you could do that. Fixed.